resources/migrations/20191023162400-permitted-roles-for-api-key.up.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE api_key ADD COLUMN permittedRoles jsonb;
```

Comment:
Okay hmm, this up-migration should probably now add all current roles as a default, right?

Reply:
that's one option yes, the other is to treat a missing permittedRoles value as "everything"